src/main/java/com/smartsheet/api/SheetResources.java

```diff
@@ -160,12 +160,12 @@ public Sheet getSheet(long id,
 	 * @throws ServiceUnavailableException if the REST API service is not available (possibly due to rate limiting)
 	 * @throws SmartsheetException if there is any other error during the operation
 	 */
-	public Sheet createSheetFromExisting(Sheet sheet, EnumSet<ObjectInclusion> includes) throws SmartsheetException;
+	public Sheet createSheetFromExisting(Sheet sheet, EnumSet<SheetTemplateInclusion> includes) throws SmartsheetException;
```

Comment:
Nice - good to see the enums broken out into call-specific sets, rather than having them shared across all calls that use Includes.